app/src/main/java/com/vci/vectorcamapp/landing/presentation/LandingViewModel.kt

```diff
@@ -23,18 +26,33 @@ class LandingViewModel @Inject constructor(
     private val deviceCache: DeviceCache,
     private val currentSessionCache: CurrentSessionCache,
     private val programRepository: ProgramRepository,
+    sessionRepository: SessionRepository,
 ) : CoreViewModel() {
 
     private val _state = MutableStateFlow(LandingState())
-    val state: StateFlow<LandingState> = _state.onStart {
-        loadLandingDetails()
+
+    private val _incompleteSessionsCount = sessionRepository.observeIncompleteSessions()
+        .map { it.size }
+        .stateIn(viewModelScope, SharingStarted.WhileSubscribed(), 0)
+
+    val state: StateFlow<LandingState> = combine(
+        _state,
+        _incompleteSessionsCount
+    ) { state, incompleteSessionsCount ->
+        state.copy(incompleteSessionsCount = incompleteSessionsCount)
     }.stateIn(
-        viewModelScope, SharingStarted.WhileSubscribed(5000L), LandingState()
+        viewModelScope,
+        SharingStarted.WhileSubscribed(),
+        LandingState()
```

Comment:
Sorry I wasnt clear in my previous comment. Typically the 5000L subscription time is required when defining the state, so you need 5000L here. Here is a video explaining this:
https://youtu.be/mNKQ9dc1knI?si=f6FTP6pXyjwW4Gcl